src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -23,17 +23,21 @@ public class ColumnMappingPresenter : AbstractPresenter<IColumnMappingView, ICol
       private UnformattedData _rawData;
       private MappingProblem _mappingProblem = new MappingProblem() { MissingMapping = new List<string>(), MissingUnit = new List<string>() };
       private readonly IMappingParameterEditorPresenter _mappingParameterEditorPresenter;
+      private readonly IMetaDataParameterEditorPresenter _metaDataParameterEditorPresenter;
       private ColumnMappingDTO _currentModel;
       public ColumnMappingPresenter
       (
          IColumnMappingView view,
          IImporter importer,
-         IMappingParameterEditorPresenter mappingParameterEditorPresenter
+         IMappingParameterEditorPresenter mappingParameterEditorPresenter,
+         IMetaDataParameterEditorPresenter metaDataParameterEditorPresenter
       ) : base(view)
       {
          _importer = importer;
          _mappingParameterEditorPresenter = mappingParameterEditorPresenter;
-         View.FillSubView(_mappingParameterEditorPresenter.BaseView);
+         _metaDataParameterEditorPresenter = metaDataParameterEditorPresenter;
+         View.FillMappingSubView(_mappingParameterEditorPresenter.BaseView);
```

Comment:
I do not think we use the term `SubView` anywhere else. Can we stick to View.
e.g. AddMappingView, AddMetaDataView etc... just for consistency sake

Reply:
@msevestre Right, fixed.